acra/acra-in-depth/architecture/acraserver.md

```diff
@@ -41,7 +41,7 @@ Your application doesn't need to handle any cryptographic code or have access to
 
 * Is easy to integrate into existing infrastructure – supports SQL databases, [supports SIEMs](/acra/security-controls/security-logging-and-events/siem-soc-integration/), [supports KMS](/acra/configuring-maintaining/key-storing/kms-integration/).
 * Is available as a package for common server Linux distros, available as Docker image. See [Getting started](/acra/getting-started/).
-* Is integrated with [AcraWebConfig](/acra/configuring-maintaining/general-configuration/acra-webconfig/) - a helper tool for re-configuring at runtime.
+* Is integrated with [AcraWebConfig](/acra/configuring-maintaining/general-configuration/acra-webconfig/) (deprecated after 0.90.0) - a helper tool for re-configuring at runtime.
```

Comment:
```suggestion
* Is integrated with [AcraWebConfig](/acra/configuring-maintaining/general-configuration/acra-webconfig/) (deprecated since 0.91.0) - a helper tool for re-configuring at runtime.
```